packages/ibm-products/src/components/Decorator/Decorator.js

```diff
@@ -0,0 +1,354 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useMemo } from 'react';
```

Comment:
i don't think anything here really _needs_ `useMemo`

Reply:
I was trying to avoid unnecessary computations on rerenders if props changed, but on reflection Decorators should not change over time. I'll remove them.

Reply:
Done.